starknet_py/tests/e2e/docs/setup/test_deploy_prefunded_account.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from starknet_py.contract import Contract
+from starknet_py.tests.e2e.conftest import MAX_FEE
+
+
+@pytest.mark.asyncio
+async def test_deploy_prefunded_account(
+    account_with_validate_deploy_class_hash: int, network: str, fee_contract: Contract
+):
+    # pylint: disable=import-outside-toplevel, too-many-locals
+    # add to docs: start
+    from starkware.crypto.signature.signature import get_random_private_key
+    from starkware.starknet.definitions.fields import ContractAddressSalt
+    from starkware.starknet.core.os.contract_address.contract_address import (
+        calculate_contract_address_from_hash,
+    )
```

Comment:
I don't like that you import things from starkware package. We for sure have `calculate_contract_address_from_hash` in our codebase. If we don't have others then we should add them.

Reply:
We've never provided functionalities like generating private keys and salt and I'm not sure if we should. 

We've always been using these methods from cairo-lang in our tests.